### DISCUSSION TITLE
Error "Invalid file" when trying to load tracks from "My collection"

### DISCUSSION THREAD

**Discussion by bios4** [devemux86/cruiser#640] [Q&A] [answered]
When trying to load tracks stored in "My collection" the error "Invalid file" is displayed and no route is calculated.

This happens in the following use cases:

- "My collection" > "Tracks / Areas" > swipe right on a stored track > Import
- "My collection" > "Tracks / Areas" > three-dots-menu on a stored track > "Calculate route" > Import
- Press navigation button > "My collection" > "Tracks / Areas" > swipe right on a stored track > Import
- Press navigation button > "My collection" > "Tracks / Areas" > three-dots-menu on a stored track > "Calculate route" > Import

When importing the very same track as GPX file from the file manager a route can be calculated without the error message!??

This happens with latest version 5.0.0-b27.

**Comment by devemux86** [comment]
@bios4 Thanks for the report.

Can you send me the file to check its content?

**Comment by devemux86** [comment]
@bios4 Thanks for the file.

I found what happens. Note: the stored files in "My collection" are OK.

The problem was in the import. It will work properly in the next update.

**Reply by bios4** [reply]
Hi Emux,

thanks for the quick analysis! 👍
I'm glad you were able to reproduce the issue...

**Reply by devemux86** [reply]
> I'm glad you were able to reproduce the issue...

This is the kind of issues that developers like to find! :slightly_smiling_face:

**Comment by devemux86** [comment]
This is fixed in Beta 28.

**Reply by bios4** [reply]
Yes, this issue seems to be gone (tested with five random routes out of "My collection") with b28!
Thank you! 👍

**Reply by devemux86** [reply]
@bios4 Thanks again for the report! :+1:

**Comment by devemux86** [comment, marked as answer]
- Fixed in https://github.com/devemux86/cruiser/discussions/641
